certify-integration-api/src/main/java/io/mosip/certify/api/spi/VCSigner.java

```diff
@@ -0,0 +1,10 @@
+package io.mosip.certify.api.spi;
+
+import io.mosip.certify.api.dto.VCResult;
+
+/**
+ * VCSigner can sign any VC provided a vcHash & Signer inputs
+ */
+public interface VCSigner {
```

Comment:
There is spi folder in both certify-core and certify-integration-api. All the interfaces are placed in these folders. So, it was placed in certify-integration-api. This can be moved to service folder of the certify-service

Reply:
VCSigner and impl file to be moved to io.mosip.certify.vcsigners